### DISCUSSION TITLE
Using it SydneyQt as api

### DISCUSSION THREAD

**Discussion by Parampatel07** [juzeon/SydneyQt#116] [General]
Can it use it as a api or on localhost for my pc

**Comment by juzeon** [comment]
Check out https://github.com/SoraRoseous/ChatSydney-react

**Reply by Parampatel07** [reply]
Okay Works well Does it has Any api Version Or Usage for it ?

**Reply by juzeon** [reply]
It's based on EdgeGPT, a project that is no longer maintained.

**Reply by Parampatel07** [reply]
ok Understandable , Thanks for the help
